domenetjenester/kobling/src/main/java/no/nav/foreldrepenger/abakus/kobling/Kobling.java

```diff
@@ -78,7 +78,7 @@ public class Kobling extends BaseEntitet implements IndexKey {
      * Inaktive koblinger skal ikke kunne endres. Det betyr oftest at koblingsreferansen er avsluttet i fagsystemet også.
      */
     @Column(name = "aktiv", nullable = false)
-    private Boolean aktiv = true;
+    private boolean aktiv = true;
```

Comment:
Ok, så vi bruker bare opp igjen dette feltet for å definere om en kobling er avsluttet eller ikke? I kalkulus er dette to ulike felter, men så tar vi jo vare på mellomlagringer i beregning. Det var kanskje slik at dette ikke ble brukt til noe ellers i abakus siden du har laget en setter under